scripts/hudson/quickstart.sh

```diff
@@ -181,16 +157,34 @@ function build_apache-karaf {
   fi
 }
 function clone_as {
-  REPO="https://github.com/wildfly/wildfly.git"
-  echo "Cloning AS sources from $REPO"
-
-  cd $WORKSPACE
+  echo "Cloning AS sources from https://github.com/jbosstm/jboss-as.git"
 
+  cd ${WORKSPACE}
   if [ -d jboss-as ]; then
     rm -rf jboss-as # start afresh
   fi
 
-  git clone $REPO jboss-as || fatal "git clone $REPO failed"
+  echo "First time checkout of WildFly"
+  git clone https://github.com/jbosstm/jboss-as.git -o jbosstm
```

Comment:
I think this block up to line 176 is going to try to check out a jbosstm/jboss-as branch main but I don't think that exists? We used to use https://github.com/jbosstm/jboss-as/tree/5_BRANCH

Reply:
That's a good point. I was going to create main in jboss-as but maybe we should use 6_BRANCH?

Reply:
https://github.com/jbosstm/jboss-as/tree/main now exists - thank you